test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -798,42 +798,13 @@ public void RestoreCommand_DisplaysCPVMInPreviewMessageIfCPVMEnabled()
                 File.WriteAllText(directoryPackagesPropsName, directoryPackagesPropsContent);
 
                 // Act
-                var result = _msbuildFixture.RunDotnet(workingDirectory, "restore /v:n");
+                var result = _msbuildFixture.RunDotnet(workingDirectory, "restore");
 
                 // Assert
                 Assert.True(result.Output.Contains($"The project {projectFile} is using CentralPackageVersionManagement, a NuGet preview feature."));
             }
         }
 
-        [PlatformFact(Platform.Windows)]
```

Comment:
Why we're removing this unit test? I feel this one compliments the above unit test.

Reply:
I don't understand the benefit of a test validating that when you don't use a feature that warnings related to that feature are not output.  In my opinion, it's like doing a restore and having tests to ensure that no pack-related warnings are output. I hope other people in the team will chime in to give their opinion.

Reply:
There's a 1 line per restore test as well, https://github.com/NuGet/NuGet.Client/blob/60b01eb4d572b91e39d35386e060167adf753d86/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs#L284.